great_expectations/experimental/datasources/schemas/SparkDatasource.json

```diff
@@ -22,10 +22,16 @@
             "additionalProperties": {
                 "$ref": "#/definitions/CSVSparkAsset"
             }
+        },
```

Comment:
@Kilo59 I may need some help next week figuring out the right way to create `SparkFilesystemDatasource`, `SparkS3Datasource`, and so on in terms of the `type` field as well as the `Schema` file naming and generation.  Most likely, this particular file will have to attain a different name (e.g., `SparkFilesystemDatasource.json`).  Thank you.